index.html

```diff
@@ -0,0 +1,316 @@
+<!DOCTYPE html>
```

Comment:
### Validate the markup
- [ ] We need to ensure that our markup does not have any informing warnings. Do you mind validating our markup?
You are free to use this [tool ](https://validator.w3.org/nu/#textarea)for easier validation.